lib/tlog/tap.c

```diff
@@ -206,21 +212,70 @@ tlog_tap_setup(struct tlog_errs **perrs,
         }
 
         /* Fork a child connected via a PTY */
-        tap.pid = forkpty(&master_fd, NULL, &tap.termios_orig, &winsize);
+        tap.pid = forkpty(&master_fd, tty_path, &tap.termios_orig, &winsize);
```

Comment:
Can we follow the warning suggestion mentioned here instead?

https://www.gnu.org/software/libc/manual/html_node/Pseudo_002dTerminal-Pairs.html#Pseudo_002dTerminal-Pairs

Reply:
The path can't be larger than PATH_MAX, so I will use it as the size of tty_path.